go/obscuronode/enclave/state.go

```diff
@@ -214,15 +216,16 @@ func FindWinner(parent *Rollup, rollups []*Rollup, s Storage, blockResolver Bloc
 	return rollups[win], true
 }
 
-func findRoundWinner(receivedRollups []*Rollup, parent *Rollup, parentState *State, s Storage, blockResolver BlockResolver) (*Rollup, *State) {
+func findRoundWinner(receivedRollups []*Rollup, parent *Rollup, parentState *State, s Storage, blockResolver BlockResolver) (*Rollup, *State, bool) {
+	var hasTxs bool
```

Comment:
@otherview Can you add a comment on the function explaining what `hasTxs` represents?

Reply:
Yep !